src/Microsoft.DotNet.ProjectJsonMigration/Rules/SaveOutputProjectRule.cs

```diff
@@ -17,6 +17,9 @@ public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migra
 
             CleanEmptyPropertyAndItemGroups(migrationRuleInputs.OutputMSBuildProject);
 
+            // Temporarily Add ProjectLockFile
+            migrationRuleInputs.CommonPropertyGroup.AddProperty("ProjectLockFile", "$(MSBuildProjectDirectory)/$(MSBuildProjectName).project.lock.json");
```

Comment:
I have this in my PR. you will see when you rebase. I added a WorkaroundRules, to be very clear that transformations there should be temporary.